orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/servergroup/DestroyServerGroupStage.groovy

```diff
@@ -36,8 +36,6 @@ class DestroyServerGroupStage extends TargetServerGroupLinearStageSupport {
   @Override
   <T extends Execution<T>> void taskGraph(Stage<T> stage, TaskNode.Builder builder) {
     try {
-      composeTargets(stage)
```

Comment:
This was just cruft from long ago. It doesn't need to do this as part of the main stage as it's done in `aroundStages`